docs/format/tile.rst

```diff
@@ -16,6 +16,15 @@ General notes:
 tilegrid.json
 -------------
 
+The file `tilegrid.json` contains lists of all tiles in the ROI and the configuration segments formed by those tiles. It also documents the membership relationship of tiles and segments.
+
+For each segment this contains the configuration frame base address, and the word offset within the frames, as well as the number of frames for the segment and number of occupied words in each frame.
```

Comment:
We should cross link to how to use the base address and word offset

Reply:
Added a FIXME.